components/dashboard/components/ui/DashBoardContentContainer.tsx

```diff
@@ -60,14 +60,31 @@ const DashBoardContentContainer = (props: DashBoardContentContainerProps) => {
   return (
     <DashBoardContentUI $isContentBar={id === '컨텐츠바'}>
       {email && <Email email={email} />}
-      {content && <Content onTitleClick={onTitleClick} content={content} />}
-      {url && <Url url={url} />}
-      {tabType && <TabType tabType={tabType} />}
-      {author && <Author author={author} />}
-      {position && <Position position={position} />}
-      {description && <Description description={description} />}
-      {draft !== undefined ? <Draft draft={draft} /> : <></>}
-      {createdAt && <CreatedAt createdAt={createdAt} />}
+      {pathName === 'page' || pathName === 'upload' || pathName === 'tempsaved' ? (
+        <PageContentWrapper $isContentBar={id === '컨텐츠바'}>
+          {content && <Content onTitleClick={onTitleClick} content={content} />}
+          {url && <Url url={url} />}
+          {tabType && <TabType tabType={tabType} />}
+          {author && <Author author={author} />}
+          {position && <Position position={position} />}
+          {description && <Description description={description} />}
+          <div className="page_content">
+            {draft !== undefined ? <Draft draft={draft} /> : <></>}
+            {createdAt && <CreatedAt createdAt={createdAt} />}
+          </div>
+        </PageContentWrapper>
+      ) : (
+        <>
+          {content && <Content onTitleClick={onTitleClick} content={content} />}
+          {url && <Url url={url} />}
+          {tabType && <TabType tabType={tabType} />}
+          {author && <Author author={author} />}
+          {position && <Position position={position} />}
+          {description && <Description description={description} />}
+          {draft !== undefined ? <Draft draft={draft} /> : <></>}
+          {createdAt && <CreatedAt createdAt={createdAt} />}
+        </>
+      )}
```

Comment:
이 중복되는 코드는 없었으면 좋겠는데,,
```suggestion
const A =  <> {content && <Content onTitleClick={onTitleClick} content={content} />}
          {url && <Url url={url} />}
          {tabType && <TabType tabType={tabType} />}
          {author && <Author author={author} />}
          {position && <Position position={position} />}
          {description && <Description description={description} />}</>;
const B = <>
            {draft !== undefined ? <Draft draft={draft} /> : <></>}
            {createdAt && <CreatedAt createdAt={createdAt} />} </>;
  return (
    <DashBoardContentUI $isContentBar={id === '컨텐츠바'}>
      {email && <Email email={email} />}
      {pathName === 'page' || pathName === 'upload' || pathName === 'tempsaved' ? (
        <PageContentWrapper $isContentBar={id === '컨텐츠바'}>
          {A}
          <div className="page_content">
               {B}
          </div>
        </PageContentWrapper>
      ) : (
        <>
         {A}
          {B}
        </>
      )}
```
이런 구조라던지.. 어떻게 생각해?

Reply:
리팩토링 진짜 여기는 해야해,,,감사합니다...
말씀하신 내용으로 수정해둘게요